DevelopGitHub/GitHubExtension/GitHubExtension.Templates/ExtensionMethods/HttpRequestMessageExtension.cs

```diff
@@ -19,14 +23,48 @@ public static class HttpRequestMessageExtension
             }
         };
 
-        public static HttpRequestMessage CreateMessage(this HttpRequestMessage message)
+        public static HttpRequestMessage
+        CreateTemplateMessage(this HttpRequestMessage requestMessage, string gitHubMessage, string content)
+        {
+            requestMessage.AddHeadersToMessage();
+
+            var final = ConvertToBase64(content);
+
+            CreateUpdateTemplateModel model = new CreateUpdateTemplateModel() { Content = final, Message = gitHubMessage };
```

Comment:
Common...Name of class HttpRequestMessageExtension but inside it using SPECIFIC CreateUpdateTemplateModel. I think it's not good.